client/src/views/Home.tsx

```diff
@@ -13,10 +13,10 @@ import { SelectChangeEvent } from '@mui/material';
 
 import type { Theme } from '@mui/material/styles';
 
-import NeedsAndOffers from './NeedsAndOffers';
-import { placeholderImg } from '../../assets/temp';
-import QuestionList from '../../assets/sharedComponents/QuestionList';
-import routes from '../../routes';
+import AssetsList from './AssetsList';
```

Comment:
AssetsList is a component that we reuse here and in multiple other places, aka not a view
let's move it to the components folder